www/class/centreonStatistics.class.php

```diff
@@ -137,6 +138,138 @@ public function getPlatformTimezone()
     }
 
+    /**
+     * get LDAP configured authentications options
+     *
+     * @return array
+     */
+    public function getLDAPAuthenticationOptions()
+    {
+        $data = [];
+
+        # Get the number of LDAP directories configured by LDAP configuration
+        $query = "SELECT ar.ar_id, COUNT(arh.auth_ressource_id) AS configured_ad
+        FROM auth_ressource_host AS arh
+        INNER JOIN auth_ressource AS ar ON (arh.auth_ressource_id = ar.ar_id)
+        WHERE ar.ar_enable = '1'
+        GROUP BY ar_id";
+        $result = $this->dbConfig->query($query);
+        while ($row = $result->fetch()) {
+            $data[$row['ar_id']] = [
+                "nb_ar_servers" => $row['configured_ad']
+            ];
+        }
+
+        # Get configured options by LDAP configuration
+        $query = "SELECT ar.ar_id, ari.ari_name, ari.ari_value
+        FROM auth_ressource_host AS arh
+        INNER JOIN auth_ressource AS ar ON (arh.auth_ressource_id = ar.ar_id)
+        INNER JOIN auth_ressource_info AS ari ON (ari.ar_id = ar.ar_id)
+        WHERE ari.ari_name IN ('ldap_template', 'ldap_auto_sync', 'ldap_sync_interval', 'ldap_auto_import',
+            'ldap_search_limit', 'ldap_search_timeout', 'ldap_srv_dns', 'ldap_store_password', 'protocol_version')";
+        $result = $this->dbConfig->query($query);
+        while ($row = $result->fetch()) {
+            $data[$row['ar_id']][$row['ari_name']] = $row['ari_value'];
+        }
+
+        return $data;
+    }
+
+    /**
+     * get Local / SSO configured authentications options
+     *
+     * @return array
+     */
+    public function getNewAuthenticationOptions()
+    {
+        $data = [];
+
+        // Get contact groups relations defined
+        $query = "SELECT COUNT(*) AS cg_relation FROM security_provider_contact_group_relation";
+        $result = $this->dbConfig->query($query);
+        $cgRelations = $result->fetchColumn();
+
+         // Get ACL groups relations defined
+        $query = " SELECT COUNT(*) AS acl_relation FROM security_provider_access_group_relation";
+        $result = $this->dbConfig->query($query);
+        $aclRelations = $result->fetchColumn();
+
+        // Get authentication configuration
+        $query = "SELECT * FROM provider_configuration WHERE is_active = '1'";
+        $result = $this->dbConfig->query($query);
+        while ($row = $result->fetch()) {
+            $customConfiguration = json_decode($row['custom_configuration'], true);
+            if ($row['type'] === 'local') {
+                $data['local'] = $customConfiguration['password_security_policy'];
+            } elseif ($row['type'] === 'web-sso') {
+                $data['web-sso'] = [
+                    'is_forced' => $row['is_forced'] ? true : false,
+                    'trusted_client_addresses'
+                        => isset($customConfiguration['trusted_client_addresses'])
+                        ? count($customConfiguration['trusted_client_addresses']) : 0,
```

Comment:
Why not this type of syntax which does not impact performance in php 7+ and seems more readable to me.
👉️ to propagate in other places if you think it useful.
```
=> count($customConfiguration['trusted_client_addresses'] ?? [])
```